File: event-service/src/main/java/com/greenlight/event/EventServiceLauncher.java
package com.greenlight.event;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@EnableR2dbcRepositories
@EnableTransactionManagement
@SpringBootApplication
public class EventServiceLauncher {
  public static void main(String[] args) {
    SpringApplication.run(EventServiceLauncher.class, args);
  }
}

File: event-service/src/main/kotlin/com/greenlight/event/EventServiceLauncher.kt
package com.greenlight.event

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@EnableDiscoveryClient
@SpringBootApplication
class EventServiceLauncher

fun main(args: Array<String>) {
    runApplication<EventServiceLauncher>(*args)
}